google/gapic/longrunning/operations_client.py

```diff
@@ -122,10 +125,17 @@ def __init__(self,
             scopes = self._ALL_SCOPES
         if client_config is None:
             client_config = {}
-        goog_api_client = '{}/{} {} gax/{} python/{}'.format(
-            app_name, app_version, self._CODE_GEN_NAME_VERSION,
-            self._GAX_VERSION, platform.python_version())
-        metadata = [('x-goog-api-client', goog_api_client)]
+
+        # Construct the correct X-Goog-API-Client header, as a string.
+        # This ends up being in the following format:
+        #   [('x-goog-api-client', 'gax/x.y.z grpc/x.y.z')]
+        #
+        # This is the format that the underlying GRPC layer expects.
+        metadata = [(
+            'x-goog-api-client',
+            metrics.stringify(metrics.fill(metrics_headers)),
```

Comment:
I don't see this in toolkit -- would very strongly prefer that this file not be hand-edited. Why does this need to be different from the normal GAPICs?